package.json

```diff
@@ -176,7 +176,7 @@
     "lint:fix": "npm run lint -- --fix",
     "maildev": "maildev",
     "postdb:migrate:dev": "npm run db:clean",
-    "postinstall": "./scripts/postinstall.sh",
+    "postinstall": "./scripts/postinstall.sh || exit 0",
```

Comment:
Why?

Reply:
was having a bug earlier this was an attempt at debugging ive removed it now